datafusion/physical-expr/src/aggregate/build_in.rs

```diff
@@ -103,16 +104,9 @@ pub fn create_aggregate_expr(
             name,
             data_type,
         )),
-        (AggregateFunction::Sum, false) => Arc::new(expressions::Sum::new(
-            input_phy_exprs[0].clone(),
-            name,
-            input_phy_types[0].clone(),
-        )),
-        (AggregateFunction::Sum, true) => Arc::new(expressions::DistinctSum::new(
-            vec![input_phy_exprs[0].clone()],
-            name,
-            data_type,
-        )),
+        (AggregateFunction::Sum, _) => {
+            return internal_err!("Builtin Sum will be removed");
+        }
```

Comment:
Hi @jayzhan211! I saw @alamb already asked the same question [here](https://github.com/apache/datafusion/pull/10651#issuecomment-2145952988) last week. I'd like to follow-up and check if you have any plans to remove this?

Reply:
@appletreeisyellow You can checkout main branch it should be removed in https://github.com/apache/datafusion/pull/10831

Reply:
@jayzhan211 Thank you so much! 💯